### PR TITLE
Remove "user" from the permission description

### LINKED ISSUE #6112: Replace "Users'" in contact module permissions
Contact module has a permission with the label `Use users' personal contact forms`. We don't have "users" in Backdrop, we have `User accounts` and `People`, we should decide which this is and adjust the label for this permission.

I am going to recommend `Use personal contact forms` without any modifier, since I think it is clear on it's own.

### PR DESCRIPTION
**Pull request by avpaderno**
Removed "user" from the permission description

Fixes https://github.com/backdrop/backdrop-issues/issues/6112

### CODE REVIEW

File: core/modules/contact/contact.module [hunk 1]
```diff
@@ -34,7 +34,7 @@ function contact_permission() {
       'title' => t('Use the site-wide contact form'),
     ),
     'access user contact forms' => array(
-      'title' => t("Use users' personal contact forms"),
+      'title' => t("Use personal contact forms"),
```
Comment (klonos) [inline core/modules/contact/contact.module]: Lets drop the double quotes now that there's not apostrophe (single quote) within the translatable string:
```suggestion
      'title' => t('Use personal contact forms'),
```

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

**Comment by klonos** [comment]
Restarting the PR sandbox...

**Comment by quicksketch** [comment]
Merged into 1.x and 1.25.x thanks @kiamlaluno!

**Pull request merged**